reconcile/ocm_upgrade_scheduler.py

```diff
@@ -87,6 +188,7 @@ def run(dry_run, gitlab_project_id=None, thread_pool_size=10):
 
     ocm_map, current_state = fetch_current_state(clusters)
     desired_state = fetch_desired_state(clusters)
+    # versions_history = get_version_history(dry_run, desired_state, ocm_map)
```

Comment:
Code and tests for `get_version_history()` look good to me.

I don't understand though how the function is intended to actually be used within the codebase. Is the versions_history output supposed to uploaded and saved somewhere?

Reply:
version_history is already uploaded to a remote state in an s3 bucket. the usage of this output is in #1795

Reply:
Got it 👍